nrf_fuel_gauge/CHANGELOG.rst

```diff
@@ -9,6 +9,29 @@ Changelog
 
 All notable changes to this project are documented on this page.
 
+Main branch
+**********************
+
+Changes
+=======
+
+* Update to library version 0.11.1.
+* The :c:func:`nrf_fuel_gauge_init` function no longer copies the battery model pointed to in :c:struct:`nrf_fuel_gauge_init_parameters`. This means that the user must ensure that the battery model is kept valid while using the library, either in ROM or RAM.
+* The :c:func:`nrf_fuel_gauge_process` function has a new argument to specify when bus voltage is present. That is, when system is powered by vbus instead of or in addition to the battery.
```

Comment:
```suggestion
* The :c:func:`nrf_fuel_gauge_init` function no longer copies the battery model pointed to in the :c:struct:`nrf_fuel_gauge_init_parameters` structure.
  This means that you must ensure that the battery model is kept valid while using the library, either in ROM or RAM.
* The :c:func:`nrf_fuel_gauge_process` function has a new argument to specify when bus voltage is present.
  That is, when system is powered by vbus instead of or in addition to the battery.
```